### PR TITLE
Redirect issues

### PR DESCRIPTION
**Pull request by tmccoy529**
Merging a beast of a branch into integration but this has all the latest and greatest and is a more stable branch than integration.

### CODE REVIEW

File: sues [hunk 1]
```diff
@@ -0,0 +1,10 @@
+  AdditionalInformationForm[m
```
Comment (martypowell) [inline sues]: is this file real?

Comment (tmccoy529) [reply sues]: hahaha what is that file? Never seen that before.

Comment (tmccoy529) [reply sues]: this has been deleted

**Comment by tmccoy529** [comment]
Yeah we have a lot to clean up. We should start logging issues in git for refactors so we don't lose anything.

**Comment by martypowell** [comment]
> Yeah we have a lot to clean up. We should start logging issues in git for refactors so we don't lose anything.

I think its understood that there will be some refactoring, not sure the benefit in logging general refactors. If it's something specific than I agree.

**Pull request merged**